src/main/java/study2/view/InputView.java

```diff
@@ -0,0 +1,32 @@
+package study2.view;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class InputView {
+
+	Scanner scanner = new Scanner(System.in);
+	private static final String PARTICIPATEMESSAGES = "참여할 사람 이름을 입력하세요. (이름은 쉼표(,)로 구분하세요)";
```

Comment:
```suggestion
	private static final String PARTICIPATE_MESSAGES = "참여할 사람 이름을 입력하세요. (이름은 쉼표(,)로 구분하세요)";
```

상수의 경우 대문자로 정의하는 컨벤션은 잘 지켜주셨는데, `_`와 같은 문자를 통해 가독성을 향상시켜주시면 좋을 것 같습니다~

```
2.11. 상수는 대문자와 언더스코어로 구성
[constant_uppercase]

상태를 가지지 않는 자료형이면서 static final`로 선언되어 있는 필드일 때를 상수로 간주한다. 상수 이름은 대문자로 작성하며, 복합어는 언더스코어(`_)를 사용하여 단어를 구분한다.

좋은 예
public final int UNLIMITED = -1;
public final String POSTAL_CODE_EXPRESSION = “POST”;
```

전체적으로 줄 바꿈, 들여쓰기 등의 컨벤션이 맞지 않는 부분이 상당히 많이 보입니다.
다음 리뷰 요청때는 개선해서 요청해주시면 좋을 것 같습니다!

네이버 핵데이에서 공개해둔 자바 코딩 컨벤션이 있어 첨부했습니다.

https://naver.github.io/hackday-conventions-java/#constant_uppercase

Reply:
컨벤션을 지키려고 할때
 이클립에서 제공하는 단축키인 ctrl + shift + f만 누름으로서 정렬해도 괜찮나요?

Reply:
이번 커밋도 전체적으로 코딩 컨벤션으로 인해
 띄어쓰기 부분에 대한 변경이력이 많을 거 같아요 ㅠㅠ